pydoctor/templatewriter/summary.py

```diff
@@ -328,19 +311,18 @@ def hasdocstring(ob):
             return True
     return False
 
-class UndocumentedSummaryPage(Element):
+class UndocumentedSummaryPage(BasePage):
+    
     filename = 'undoccedSummary.html'
```

Comment:
This output file is generated, but nothing links there. It's fine to leave it as-is for this PR, but we should think about whether this page is useful.

It does look slightly broken at the moment:
- the public/private filter doesn't work: private names are always shown
- inherited methods without a docstring are listed, even if the method they inherit is documented